Resources/Prototypes/Roles/Jobs/Engineering/atmospheric_technician.yml

```diff
@@ -1,6 +1,6 @@
 - type: job
   id: AtmosphericTechnician
-  name: "Atmospheric Technician"

```

Comment:
This is a UI bug and I don't think this needs shortened as this changes it in-game too.

Reply:
ok ill just uncapitalize it then so it maches the others